### ISSUE TITLE
MD077: continuation in unordered lists are checked only in tight continuation

### ISSUE THREAD

**Issue by alfrisch** [rvben/rumdl#653]
### rumdl version

0.2.5

### Installation method

cargo install

### Bug description

Rule MD077 (https://github.com/rvben/rumdl/blob/1fc3ed56ecea18af3ab2221c465be2a76e315a9b/docs/md077.md) ensures that list continuation text is correctly indented.

However, list continuation text is only checked and formatted correctly when tightly continued, a blank line between the list item and the text causes the rule to not trigger.

In tight continuation, if the continuation extends over several blocks the check stops working after the first block too, due to the blank line.

### Steps to reproduce

1. Create a Markdown file with this content:
    ```markdown
    ---
    title: Heading
    ---
    
    Some introductory text:
    
    * This is a list item.
       This is list continuation text and
      it has multiple lines that aren't indented properly.
       This is yet another line that isn't indented properly.
    
    * This is a list item.
    
       This is list continuation text and
      it has multiple lines that aren't indented properly.
       This is yet another line that isn't indented properly.
   ```
1. Run `rumdl check` on Markdown file. See that line 8 and line 10 are flagged.
1. Run `rumdl fmt` on Markdown file. Take note of result.

### Expected behavior

Given the scenario above, I'd expect the Markdown to be transformed into:

```markdown
---
title: Heading
---

Some introductory text:

* This is a list item.
  This is list continuation text and
  it has multiple lines that aren't indented properly.
  This is yet another line that isn't indented properly.

* This is a list item.

  This is list continuation text and
  it has multiple lines that aren't indented properly.
  This is yet another line that isn't indented properly.
```

### Actual behavior

Given the scenario above, the Markdown is transformed into the following:

```markdown
---
title: Heading
---

Some introductory text:

* This is a list item.
  This is list continuation text and
  it has multiple lines that aren't indented properly.
  This is yet another line that isn't indented properly.

* This is a list item.

   This is list continuation text and
  it has multiple lines that aren't indented properly.
   This is yet another line that isn't indented properly.
```

### Configuration

```toml

```

### Operating system

macOS

### Additional context

```
$ rumdl config --no-defaults
[global]                                                
disable = ["MD013", "MD034", "MD040", "MD041", "MD046"] [from pyproject.toml]
include = ["changelog.d/", "changelogs"]                [from pyproject.toml]

[per-file-ignores]                                      
"changelog.d/template.md" = ["MD022", "MD032"]          [from pyproject.toml]
```

**Comment by rvben** [comment]
Fixed in [v0.2.7](https://github.com/rvben/rumdl/releases/tag/v0.2.7). Over-indented continuation is now normalized after a blank line too.

Thanks for the clear report and reproducer, @alfrisch!
